app/src/main/java/live/teekamsuthar/mutify/MainActivity.java

```diff
@@ -743,4 +749,59 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
             }
         }
     }
+
+    /* check if the device is chinese brand
+        if yes, then show up a dialog which will navigate user to the auto start settings in his phone
+        where he/she can mark this app to run in the background (because chinese brand device don't let apps
+        to run foreground notification in the background until user manually give permission for it)
+     */
+    private void isChineseBrand(){
+        String device = Build.MANUFACTURER.toLowerCase();
+        Intent intent;
+        switch (device){
+            case "xiaomi":
```

Comment:
Can we also add this check for Samsung devices here? I am getting frequent reports of background service getting killed from Samsung users.

Reply:
sure let me update the code  and  send another pull request